tests/kernel/nfs_stress_ng.pm

```diff
@@ -15,6 +15,32 @@ use utils;
 use registration;
 use version_utils 'is_sle';
 
+sub check_nfs_mounts {
+    my ($paths_ref) = @_;
```

Comment:
You can do `my @paths = @_;` and then call the function as `check_nfs_mounts(@paths);`